uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.spec.js

```diff
@@ -156,34 +156,34 @@ define([
       });
 
       it('removes assignee form the list of assignees', function () {
```

Comment:
Not a nice test. `expect` should only contain `it`s. Please amend the test (you may want to create a new `describe` with beforeEach)